apis/v1/gateway_types.go

```diff
@@ -303,6 +303,16 @@ type GatewaySpec struct {
 	// +optional
 	// <gateway:experimental>
 	TLS *GatewayTLSConfig `json:"tls,omitempty"`
+
+	// DefaultScope defines the default scope for this Gateway. If unset (the
+	// default), the Gateway will not act as a default Gateway; if set, the
```

Comment:
```suggestion
	// DefaultScope defines the default scope for this Gateway. If unset or
	// set to the empty string `""` (the default), the Gateway will not
	// act as a default Gateway; if set, the
```

I know, I know, it's very specific, but not being specific enough has caught us enough times that I'd rather err on the side of over-specifying than under-specifying.

Reply:
Yeah, that's a nice catch. 👍